### ISSUE TITLE
Registrace

### ISSUE THREAD

**Issue by kubaran11** [Stredni-skola-managementu-a-grafiky/UP3AI#21]
Zpracovat ověřenou registraci uživatele, připravit propojení registrace do DB.

**Comment by HavelJakub** [comment]
html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Registrace</title>
    <link rel="stylesheet" href="stylesheet.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script> 
    
    
</head>

<body>
    <form class="well form-horizontal" action=" " method="post"  id="contact_form">
        <fieldset>
        
        
        <legend><center><h2><b>Registration Form</b></h2></center></legend><br>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label">Jméno</label>  
          <div class="col-md-4 inputGroupContainer">
          <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input  name="first_name" placeholder="Jméno" class="form-control"  type="text">
            </div>
          </div>
        </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Přijmení</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="last_name" placeholder="Přijmení" class="form-control"  type="text">
            </div>
          </div>
        </div>

        
        <div class="form-group">
            <label class="col-md-4 control-label">E-Mail</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="email" placeholder="E-Mail" class="form-control"  type="text">
              </div>
            </div>
          </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Heslo</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="password1" placeholder="Heslo" class="form-control"  type="password">
            </div>
          </div>
        </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Potvrdit heslo</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="password2" placeholder="Potvrdit heslo" class="form-control"  type="password">
            </div>
          </div>
        </div>
        
        <div class="form-group">
            <label class="col-md-4 control-label">Město</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="city" placeholder="" class="form-control" type="text">
              </div>
            </div>
          </div>

          <div class="form-group">
            <label class="col-md-4 control-label">Ulice</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="street" placeholder="" class="form-control" type="text">
              </div>
            </div>
          </div>

          <div class="form-group">
            <label class="col-md-4 control-label">Číslo popisné</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon" ></span>
            <input name="housenum placeholder="" class="form-control" type="text">
            
              </div>
            </div>
          </div>
        
               
        <div class="form-group">
          <label class="col-md-4 control-label">Tel.číslo</label>  
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
                <span class="input-group-addon"><i class="glyphicon glyphicon-earphone"></i></span>
          <input name="contact_no" placeholder="(+420)" class="form-control" type="text">
            </div>
          </div>
        </div>
        
        
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label"></label>
          <div class="col-md-4"><br>
            &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp<button type="submit" class="btn-grad" >&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbspZaregistrovat <span class="glyphicon glyphicon-send"></span>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</button>
          </div>
        </div>
        </fieldset>
        </form>
        </div>
            </div>

    <script src="register.js"></script> 
</body>
</html>

**Comment by HavelJakub** [comment]
`<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Registrace</title>
    <link rel="stylesheet" href="stylesheet.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script> 
    
    
</head>

<body>
    <form class="well form-horizontal" action=" " method="post"  id="contact_form">
        <fieldset>
        
        
        <legend><center><h2><b>Registration Form</b></h2></center></legend><br>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label">Jméno</label>  
          <div class="col-md-4 inputGroupContainer">
          <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input  name="first_name" placeholder="Jméno" class="form-control"  type="text">
            </div>
          </div>
        </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Přijmení</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="last_name" placeholder="Přijmení" class="form-control"  type="text">
            </div>
          </div>
        </div>

        
        <div class="form-group">
            <label class="col-md-4 control-label">E-Mail</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="email" placeholder="E-Mail" class="form-control"  type="text">
              </div>
            </div>
          </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Heslo</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="password1" placeholder="Heslo" class="form-control"  type="password">
            </div>
          </div>
        </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Potvrdit heslo</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="password2" placeholder="Potvrdit heslo" class="form-control"  type="password">
            </div>
          </div>
        </div>
        
        <div class="form-group">
            <label class="col-md-4 control-label">Město</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="city" placeholder="" class="form-control" type="text">
              </div>
            </div>
          </div>

          <div class="form-group">
            <label class="col-md-4 control-label">Ulice</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="street" placeholder="" class="form-control" type="text">
              </div>
            </div>
          </div>

          <div class="form-group">
            <label class="col-md-4 control-label">Číslo popisné</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon" ></span>
            <input name="housenum placeholder="" class="form-control" type="text">
            
              </div>
            </div>
          </div>
        
               
        <div class="form-group">
          <label class="col-md-4 control-label">Tel.číslo</label>  
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
                <span class="input-group-addon"><i class="glyphicon glyphicon-earphone"></i></span>
          <input name="contact_no" placeholder="(+420)" class="form-control" type="text">
            </div>
          </div>
        </div>
        
        
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label"></label>
          <div class="col-md-4"><br>
            &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp<button type="submit" class="btn-grad" >&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbspZaregistrovat <span class="glyphicon glyphicon-send"></span>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</button>
          </div>
        </div>
        </fieldset>
        </form>
        </div>
            </div>

    <script src="register.js"></script> 
</body>
</html>`

**Comment by HavelJakub** [comment]
"<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Registrace</title>
    <link rel="stylesheet" href="stylesheet.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script> 
    
    
</head>

<body>
    <form class="well form-horizontal" action=" " method="post"  id="contact_form">
        <fieldset>
        
        
        <legend><center><h2><b>Registration Form</b></h2></center></legend><br>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label">Jméno</label>  
          <div class="col-md-4 inputGroupContainer">
          <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input  name="first_name" placeholder="Jméno" class="form-control"  type="text">
            </div>
          </div>
        </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Přijmení</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="last_name" placeholder="Přijmení" class="form-control"  type="text">
            </div>
          </div>
        </div>

        
        <div class="form-group">
            <label class="col-md-4 control-label">E-Mail</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="email" placeholder="E-Mail" class="form-control"  type="text">
              </div>
            </div>
          </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Heslo</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="password1" placeholder="Heslo" class="form-control"  type="password">
            </div>
          </div>
        </div>
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label" >Potvrdit heslo</label> 
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
          <span class="input-group-addon"><i class=""></i></span>
          <input name="password2" placeholder="Potvrdit heslo" class="form-control"  type="password">
            </div>
          </div>
        </div>
        
        <div class="form-group">
            <label class="col-md-4 control-label">Město</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="city" placeholder="" class="form-control" type="text">
              </div>
            </div>
          </div>

          <div class="form-group">
            <label class="col-md-4 control-label">Ulice</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon"><i class=""></i></span>
            <input name="street" placeholder="" class="form-control" type="text">
              </div>
            </div>
          </div>

          <div class="form-group">
            <label class="col-md-4 control-label">Číslo popisné</label>  
              <div class="col-md-4 inputGroupContainer">
              <div class="input-group">
                  <span class="input-group-addon" ></span>
            <input name="housenum placeholder="" class="form-control" type="text">
            
              </div>
            </div>
          </div>
        
               
        <div class="form-group">
          <label class="col-md-4 control-label">Tel.číslo</label>  
            <div class="col-md-4 inputGroupContainer">
            <div class="input-group">
                <span class="input-group-addon"><i class="glyphicon glyphicon-earphone"></i></span>
          <input name="contact_no" placeholder="(+420)" class="form-control" type="text">
            </div>
          </div>
        </div>
        
        
        
        
        
        <div class="form-group">
          <label class="col-md-4 control-label"></label>
          <div class="col-md-4"><br>
            &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp<button type="submit" class="btn-grad" >&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbspZaregistrovat <span class="glyphicon glyphicon-send"></span>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</button>
          </div>
        </div>
        </fieldset>
        </form>
        </div>
            </div>

    <script src="register.js"></script> 
</body>
</html>"

**Comment by matysko11** [comment]
[login.txt](https://github.com/Stredni-skola-managementu-a-grafiky/UP3AI/files/8608854/login.txt)

**Comment by matysko11** [comment]
[index.txt](https://github.com/Stredni-skola-managementu-a-grafiky/UP3AI/files/8608856/index.txt)

**Comment by HavelJakub** [comment]
[stylesheet registrace.txt](https://github.com/Stredni-skola-managementu-a-grafiky/UP3AI/files/8608860/stylesheet.registrace.txt)
[Registrace html.txt](https://github.com/Stredni-skola-managementu-a-grafiky/UP3AI/files/8608861/Registrace.html.txt)

**Comment by matysko11** [comment]
![pozadíRegistr](https://user-images.githubusercontent.com/102223073/166417042-850978e6-6472-41a3-a171-d252bc52b82d.png)

**Comment by matysko11** [comment]
[css.txt](https://github.com/Stredni-skola-managementu-a-grafiky/UP3AI/files/8608868/css.txt)

**Comment by kubaran11** [comment]
@HavelJakub Případně přesmerovat na #36.
